aptos-move/framework/aptos-framework/sources/transaction_context.spec.move

```diff
@@ -11,9 +14,26 @@ spec aptos_framework::transaction_context {
         ensures result == spec_get_txn_hash();
     }
     spec fun spec_get_txn_hash(): vector<u8>;
+    spec get_transaction_hash(): vector<u8> {
+        pragma opaque;
+        // property 1: Fetching the transaction hash should return a vector with 32 bytes, if the auid feature flag is enabled.
+        aborts_if [abstract] !std::features::auids_enabled();
+        ensures [abstract] len(result) == 32;
+    }
     spec generate_unique_address(): address {
         pragma opaque;
         ensures [abstract] result == spec_generate_unique_address();
     }
     spec fun spec_generate_unique_address(): address;
+    spec generate_auid_address(): address {
+        pragma opaque;
+        // property 3: Generating the unique address should return a vector with 32 bytes, if the auid feature flag is enabled.
+        aborts_if [abstract] !std::features::auids_enabled();
+        ensures [abstract] result == spec_generate_unique_address();
+        // ensures [abstract] len(result) == 32;
```

Comment:
why is this commented out?

Reply:
It‘s because the address cannot be verify as an vector, deleted